dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -516,6 +586,48 @@ def _perform_t_test(
         results["welch"]["p-value"] = float(welch_p_val)
         return results
 
+    @staticmethod
+    def _calculate_psi(
+        mean1: float,
+        sum1: float,
+        histogram1: np.ndarray,
+        mean2: float,
+        sum2: float,
+        histogram2: np.ndarray,
+    ) -> Optional[float]:
+        self_count = sum1 / mean1
+        other_count = sum2 / mean2
```

Comment:
If the intent is to get the count, we could just sum the bin_counts instead. otherwise we might lose floating point accuracies, no? Also:
```python
def mean(self) -> float:
        """Return mean value."""
        if self.match_count == 0:
            return 0
        return float(self.sum) / self.match_count
```
so could just use `match_count` as well.

Reply:
Perf! I'll use `match_count` -- less code and reuse what already exists